### ISSUE TITLE
K2 incompatible

### ISSUE THREAD

**Issue by eslamwael74** [pszklarska/FlutterPubVersionChecker#65]
The plugin is incompatible with the Kotlin plugin in k2 mode
![Screenshot 2024-08-26 at 2 19 19 PM](https://github.com/user-attachments/assets/c46ca051-8ad4-43f6-9b66-6c7495966921)

**Comment by peacemaker123456** [comment]
yes please fix, K2 will become the default in coming intellij version

**Comment by pszklarska** [comment]
Thanks for the information, I'll look into this soon 👍
